pkg/crd/markers/validation.go

```diff
@@ -83,8 +83,10 @@ var FieldOnlyMarkers = []*definitionWithHelp{
 		WithHelp(markers.SimpleHelp("CRD validation", "specifies that this field is required, if fields are optional by default.")),
 	must(markers.MakeDefinition("kubebuilder:validation:Optional", markers.DescribesField, struct{}{})).
 		WithHelp(markers.SimpleHelp("CRD validation", "specifies that this field is optional, if fields are required by default.")),
+	must(markers.MakeDefinition("required", markers.DescribesField, struct{}{})).
+		WithHelp(markers.SimpleHelp("CRD validation", "specifies that this field is required.")),
 	must(markers.MakeDefinition("optional", markers.DescribesField, struct{}{})).
-		WithHelp(markers.SimpleHelp("CRD validation", "specifies that this field is optional, if fields are required by default.")),
```

Comment:
Docs change not behavioural change right?

Reply:
Can see from changes below this is just a docs change, LGTM

Reply:
@liggitt Should we also drop this "if" for the kubebuilder markers above?

Basically whenever any of the markers are set they are authoritative, doesn't matter what the default is

Reply:
Yes, the markers are authoritative. Made all docs consistent.